cmd/erasure-sets.go

```diff
@@ -738,31 +739,45 @@ func (s *erasureSets) ListBuckets(ctx context.Context) (buckets []BucketInfo, er
 	return listBuckets, nil
 }
 
+func logBackendServers(ctx context.Context, endpoints []string) {
```

Comment:
```suggestion
func setBackendServersToLog(ctx context.Context, endpoints []string) {
```

Reply:
Perhaps we should not do this computation all the time @vadmeste using `strings.Join()` for memory - we should store the entire string of backendServers per `cmd/erasure.go`  just like `getEndpoints()`

Reply:
> Perhaps we should not do this computation all the time @vadmeste using `strings.Join()` for memory - we should store the entire string of backendServers per `cmd/erasure.go` just like `getEndpoints()`

Done